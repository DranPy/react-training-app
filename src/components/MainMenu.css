.main-menu {
  display: flex
}

.main-menu .status-bar {
  margin-left: auto;
}

.nav .nav-link {
  color: #fff;
  border-bottom: 1px solid transparent;
}

.nav .nav-link:hover {
  border-bottom-color: #ff000075;
}

.nav .nav-link.active {
  border-bottom-color: red;
}