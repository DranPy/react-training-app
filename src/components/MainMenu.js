import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

import './MainMenu.css';

class MainMenu extends Component {
  render() {
    return (
      <div className="main-menu">
        <ul className="nav nav--left">
          <li>
            <NavLink className="nav-link" to="/products">
              Products
            </NavLink>
          </li>
        </ul>
        <ul className="nav nav--right">
          <li className="nav-item">
            <NavLink className="nav-link" to="/login">
              Login
            </NavLink>
          </li>
          <li className="nav-item">
            <NavLink className="nav-link" to="/register">
              Register
            </NavLink>
          </li>
        </ul>
      </div>
    );
  }
}

export default MainMenu;
