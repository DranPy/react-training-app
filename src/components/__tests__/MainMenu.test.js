import { shallow } from 'enzyme';
import React from 'react';

import MainMenu from '../MainMenu';

describe('<MainMenu />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<MainMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
