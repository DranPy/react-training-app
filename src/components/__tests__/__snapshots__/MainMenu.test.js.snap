// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainMenu /> should match snapshot 1`] = `
<div
  className="main-menu"
>
  <ul
    className="nav nav--left"
  >
    <li>
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="nav-link"
        to="/products"
      >
        Products
      </NavLink>
    </li>
  </ul>
  <ul
    className="nav nav--right"
  >
    <li
      className="nav-item"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="nav-link"
        to="/login"
      >
        Login
      </NavLink>
    </li>
    <li
      className="nav-item"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="nav-link"
        to="/register"
      >
        Register
      </NavLink>
    </li>
  </ul>
</div>
`;
